components/brave_perf_predictor/browser/bandwidth_savings_predictor_unittest.cc

```diff
@@ -91,7 +92,7 @@ TEST_F(BandwidthSavingsPredictorTest, FeaturiseResourceLoading) {
 
   auto tp_style = predictors::CreateResourceLoadInfo(
       "https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.js",
-      blink::mojom::ResourceType::kScript);
+      network::mojom::RequestDestination::kScript);
```

Comment:
any header/dep changes for this?

Reply:
Already has the header and dep via //brave/components/brave_perf_predictor/browser